custom_messages/msg/FollowerCommand.msg

```diff
@@ -0,0 +1,7 @@
+string  commandType
+string  actionOnDone
+string  strParam1
+string  strParam2
+float64 numParam1
+float64 numParam2
```

Comment:
These field names are not very descriptive.

Reply:
The point is it is a generic command block defined by the command type.  If you take a look at the Readme you will gain insight into its need to be non descriptive.

Reply:
That makes sense